src/boardgameprojectfile/boardgameprojectfile.cpp

```diff
@@ -0,0 +1,38 @@
+#include "boardgameprojectfile.h"
+#include <QFile>
+#include <QDataStream>
+#include <QString>
+#include <QDebug>
+
+#include "bgame_generated.h"
+
+std::unique_ptr<BoardGame> BoardGameProjectFile::load(const QString &path)
+{
+    //this method is intentionally empty and will be implemented later on
+}
+
+void BoardGameProjectFile::save(const QString &path)
+{
+    // Build up a serialized buffer algorithmically:
+    flatbuffers::FlatBufferBuilder builder;
+
+    // First, lets serialize some description.
+    auto descriptionOne = builder.CreateString("Orest");
+
+    // Use the `CreateBoardGame` shortcut to create BoardGame with all fields set.
+    auto orest = CreateBoardGame(builder, descriptionOne);
+
+    builder.Finish(orest); // Serialize the root of the object.
+
+    QFile file("file.bgame");
+    if(file.open(QIODevice::WriteOnly))
+    {
+        QDataStream stream(&file);
+        stream << builder.GetBufferPointer() << builder.GetSize();
+        if(stream.status() != QDataStream::Ok)
+        {
+            qDebug() << "Error";

```

Comment:
Please, remove this debug output. 
Also if we need to handle errors - we should do it in more accurate way